shared/test-helpers/ganache-cli.sh

```diff
@@ -30,8 +30,11 @@ fi
 
 echo "Starting our own testrpc instance at port $testrpc_port"
 start_testrpc
+sleep 5
 
-if [ "$TRUFFLE_TEST" = true ]; then
+if [ "$SOLIDITY_COVERAGE" = true ]; then
+  ./node_modules/.bin/solidity-coverage "$@"
+elif [ "$TRUFFLE_TEST" = true ]; then
   truffle test --network rpc "$@"
-  kill -9 $testrpc_pid
 fi
+kill -9 $testrpc_pid
```

Comment:
@bingen I'm pretty sure this should also kill the solidity-coverage testrpc instance?

Reply:
I don't think so, as we have `set -o errexit` at the beginning, therefore if a process returns error it will exit before.

Reply:
Btw, were you replying to this? ;-)
https://github.com/aragon/aragonOS/pull/291#issuecomment-381430736
It's in a different repo, but the same problem.

Reply:
Something different; was wondering if we're correctly capturing the testrpc instance solidity-coverage starts in `$testrpc_pid` (I don't see why it wouldn't, but you know solidity-coverage...)